### ISSUE TITLE
`vargo new --neon`

### ISSUE THREAD

**Issue by dherman** [neon-bindings/rfcs#8]
I'd like to see a proposal for a command-line flag for `vargo new` that lets you specify a local path to a local development tree of `neon`, so that you can easily build neon projects with a custom build or fork of neon. This would be useful in particular for testing.

**Comment by nixpulvis** [comment]
I'm a little confused by the naming here. `vargo` is going to be the new name of the project, then why are we still referencing neon. I'd expect this flag to be called something like `--src` or `--use-local`... There's a question of what exactly can be referenced, for example maybe we want to allow `--use=git@github.com:<username>/neon.git` or other protocols. Food for thought.

Maybe I miss-understood, and neon will remain the name of project, and `vargo` will just be the CLI. Gotta love renaming things 😈

**Comment by dherman** [comment]
> Maybe I miss-understood, and neon will remain the name of project, and vargo will just be the CLI. 

Right. The project is still and will always be neon. `vargo` is a temporary name for the CLI, until we can get to a point where `cargo` has enough extension hooks and we don't need a wrapper script at all.
